src/core/viz/expressions/binary.js

```diff
@@ -215,12 +287,21 @@ export const GreaterThanOrEqualTo = genBinaryOp('greaterThanOrEqualTo',
  * @param {carto.expressions.Base|number} y - Numeric expression
  * @return {carto.expressions.Base} Numeric expression
  *
- * @example <caption>Compare two numbers to show only elements with price < 30</caption>
+ * @example <caption>Compare two numbers to show only elements with price lower than 30.</caption>
```

Comment:
grammar police again: is lower OK? (usually "less" is used)

Reply:
mmm 🤔 for "price" it sounds right, maybe the grammar police was too zealous here